Update manifest: Vendicated.Vencord version 1.4.0

Dragon1573 opened the pull request:
## What's changed?

- Bump manifest version: `1.5.0` -> `1.10.0`
- Update installer hash value
- Prettify some properties

## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #262961 

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## Manual validation

> [!WARNING]
> 
> The first black console will automatically pop up during installation. It might block the "unattended" install process of the package and might cause https://github.com/microsoft/winget-pkgs/labels/Validation-Unattended-Failed.

![Image](https://github.com/user-attachments/assets/cd3d7e21-31e2-49c4-81a8-72f1358f6acf)
![Image](https://github.com/user-attachments/assets/8ea0a685-a292-4c7a-bc51-f97f4057599c)
![Image](https://github.com/user-attachments/assets/2e6f374f-93b1-448a-9fd7-77a803b6a1e9)

> [!IMPORTANT]
> 
> Users HAVE TO login to _Discord Client_ to see effects of _Vencord_.

-----


 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/264074)

Dragon1573 committed:
Update manifest: Vendicated.Vencord version 1.4.0

- Bump manifest version: `1.5.0` -> `1.10.0`
- Update installer hash value
- Prettify some properties

stephengillie commented:
Automatic Validation ended with:
>  Installation failed with exit code -1978334972 2025-06-09 06:04:10.601 [CLI ] No suitable installer found for manifest Discord.Discord with version 1.0.9194
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | 8A150104 | -1978334972 | 2316632324 | APPINSTALLER_CLI_ERROR_INSTALL_MISSING_DEPENDENCY | This package has a dependency missing from your system. (The most common dependency is `Microsoft.VCRedist.2015+.x64` / `.x86`) | 


(Automated response - build 1064.)

Dragon1573 commented:
> Automatic Validation ended with:
> 
> ```text
> Installation failed with exit code -1978334972 2025-06-09 06:04:10.601 [CLI ] No suitable installer found for manifest Discord.Discord with version 1.0.9194
> ```

During manual validation, NO extra _Visual C++ Runtime_ is required. 😕

Dragon1573 commented on manifests/v/Vendicated/Vencord/1.4.0/Vendicated.Vencord.installer.yaml:
```diff
@@ -12,11 +12,11 @@ InstallerSwitches:
+ReleaseDate: 2024-02-24
+RequireExplicitUpgrade: true
 Installers:
 - Architecture: x86
```
```suggestion
- Architecture: x64
```

> [!IMPORTANT]
> 
> It seems there's NO `x86` arch distributions of `Discord.Discord` available.

```text
23:09:54 D:\...\winget-pkgs  [issues/262963 ≡] 0ms pwsh
$ komac show --installer-manifest Discord.Discord -v 1.0.9194 |
> ConvertFrom-Yaml |
> Select-Object -ExpandProperty Installers |
> Format-List

Name  : InstallerUrl
Value : https://stable.dl2.discordapp.net/distro/app/stable/win/x64/1.0.9194/DiscordSetup.exe

Name  : InstallerSha256
Value : C1C9A01D72966CB9CEC73B9AE53660B4FD98D80CF288B83412003481BC166DFC

Name  : Architecture
Value : x64
```

Dragon1573 committed:
Apply suggestions from code review

- Switch installer architecture: `x86` -> `x64`
  - No available `x86` build of Discord

stephengillie commented:
Automatic Validation ended with:
> No errors to post.

(Automated response - build 1064.)
